shared/oval/accounts_max_concurrent_login_sessions.xml

```diff
@@ -54,7 +54,7 @@
     <ind:path>/etc/security/limits.d</ind:path>
     <ind:filename operation="pattern match">^.*\.conf$</ind:filename>
     <ind:pattern operation="pattern match">^[\s]*\*[\s]+(?:(?:hard)|(?:-))[\s]+maxlogins</ind:pattern>
-    <ind:instance datatype="int">1</ind:instance>
+    <ind:instance datatype="int" operation="greater than or equal">0</ind:instance>
```

Comment:
Hmmm.  Doesn't look like you're missing anything.

I was playing with the idea of removing the negate in the criterion.  That would have made this "operation="equal">0<

My best guess right now is that when I backed that out, I forgot to change the number back to 1, and only changed the operation.  I do recall testing it after having done that, and the tests doing what I expected.  But perhaps I made a mistake in that testing.

I'll test it again with the current code.  If that passes, I'll say so.  Otherwise (expected), I'll upload a revision.